docs/configuration/configuration-file.mdx

```diff
@@ -5,224 +5,453 @@ import { StyledContent } from '../../components/styled.elements';
-    features.openapi: []
+      - main
+    lint:
+      rules:
+        example-rule: warn
+```
+
+Not all `lint` options are supported for individual APIs.
+
+Option | Per-API support | Behavior
+-- | -- | --
+lint.extends | ❗ | API settings override global settings.
+lint.rules | ✅ | API and global settings merge. In case of conflict, API takes priority.
+lint.decorators | ✅ | API and global settings merge. In case of conflict, API takes priority.
+lint.preprocessors | ✅ | API and global settings merge. In case of conflict, API takes priority.
+lint.plugins | ❌ | Available in global configuration only.
+
+#### The `features.openapi` property
+The global (top-level) `features.openapi` section is where you configure how all of your published reference docs will look (style, fonts, colors — known as *theming*) and behave (enable/disable search, show/hide API reference docs download option — known as *functionality*).
+
+If you need to apply different theming and functionality to individual APIs, just add the `features-openapi` property to the `apis` section, and use the same options as the global `features-openapi`.
+
+Option | Per-API support | Behavior
+-- | -- | --
+features.openapi | ✅ | API and global settings merge. In case of conflict, API takes priority.
+
+#### The `features-mockServer` property
+You can apply `features.mockServer` to individual APIs as well as at the global level.
+
+Option | Per-API support | Behavior
+-- | -- | --
+features.mockServer | ✅ | API and global settings merge. In case of conflict, API takes priority.
+
+## `lint`
+
+### Overview
+The `lint` section lets you set rules that control how the `lint` and `bundle` commands behave.
+
+You can set global (top-level) rules for all OpenAPI definition files, or for specific files listed in the `apis` section. If `lint` rules are defined for individual APIs, their settings apply *together* with the global configuration. So, if API and global sections modify the same properties, API settings will always override global settings.
+
+The `lint` section is broken into smaller sub-sections, each defining different configuration options. Here's an example showing the `plugins`, `extends` and `rules` sub-sections:
+
+```yaml
 lint:
+  plugins:
+    - './local-plugin.js'
+  extends:
+    - recommended
   rules:
-    example-rule-name: error
-features.openapi:
-  pagination: section
-  showConsole: true
-  theme:
-    menu:
-      backgroundColor: '#fffff'
+    no-sibling-refs:
+      severity: error
+    boolean-parameter-prefixes:
+      severity: error
+      prefixes: ['should', 'is', 'has']
+  ...
 ```
 
+Each sub-section is described below. All are optional, so only add them to the `lint` section if needed.
+
+:::info
+The `rules` and `decorators` sub-sections only work at the API level. The `plugins` sub-section only works at the global level.
+:::
 
-The `apis` section supports several optional properties for every listed API.
+#### The `plugins` sub-section
```

Comment:
question: Is there a reason for it to not be "subsection"?

Reply:
No reason. It's just how I have expressed this word in the past. I'll use subsection from now on.